src/sql/src/session/vars/definitions.rs

```diff
@@ -541,6 +541,13 @@ pub static MAX_ROLES: VarDefinition = VarDefinition::new(
     true,
 );
 
+pub static MAX_CONTINUAL_TASKS: VarDefinition = VarDefinition::new(
+    "max_continual_tasks",
+    value!(u32; 100),
+    "The maximum number of continual_tasks in the region, across all schemas (Materialize).",
+    true,
```

Comment:
This makes the existence of CTs externally visible outside of mz_internal, but that seems better than me forgetting to flip it to public later 🤷 (Also, trying hard to not accumulate more TODO(ct)s, I have plenty)